recipes/tini/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+glibc-devel
+glibc-static
```

Comment:
Only need `glibc-static` as the static build of `tini` always happens currently. If we are able to disable the static build in the future, we can then drop `glibc-static`.

xref: xref: https://github.com/krallin/tini/issues/93